ec2/ssh/Vagrantfile

```diff
@@ -0,0 +1,222 @@
+require 'json'
```

Comment:
why do we need this ssh/Vagrantfile?

Reply:
as a backup or for anyone that's using some other type of cloud (Azure, Tata, GCP, etc). It's a generic linux setup

Reply:
sweet!